modules/@apostrophecms/area/ui/apos/lib/get-widgets.js

```diff
@@ -0,0 +1,14 @@
+// Requires an area field's .options property
+
+export default function getWidgets(options) {
```

Comment:
I didn't strictly have to factor this out today, but in the long run I'd like to have one implementation, so this is the right direction.